lib/sqlitex/server.ex

```diff
@@ -15,6 +15,10 @@ defmodule Sqlitex.Server do
   {:ok, [[a: 2, b: 2]]}
   iex> Sqlitex.Server.query(:example, "SELECT * FROM t ORDER BY a LIMIT 1", into: %{})
   {:ok, [%{a: 1, b: 1}]}
+  iex> Sqlitex.Server.query(:example, "SELECT * FROM t ORDER BY a LIMIT 2", into: :raw_list)
+  {:ok, [[1, 1], [2, 2]], [:a, :b]}
+  iex> Sqlitex.Server.query(:example, "SELECT * FROM t ORDER BY a LIMIT 2", into: :raw_list, types: true)
+  {:ok, [[1, 1], [2, 2]], [:a, :b], [:INTEGER, :INTEGER]}
```

Comment:
I was thinking that the function would return a map like:

```elixir
{:ok, %{rows: [...], columns: [...], types: [...]}
```

That would obviously be a breaking change so maybe we would introduce it under a new function name and keep the old behavior under the existing name.